A long cat is long.py

```diff
@@ -0,0 +1,9 @@
+def clean_word(word):
+     return list(filter(lambda x:x.isalpha(),word))
+
+def  count_words(text):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/